package.json

```diff
@@ -2,11 +2,14 @@
   "name": "episode-parser",
   "version": "1.2.0",
   "description": "Parse file names of tv show episodes.",
-  "main": "lib/parser.js",
+  "main": "dist/parser.js",
   "scripts": {
     "style": "./node_modules/.bin/eslint .",
     "test": "npm run style && npm run unittest",
-    "unittest": "./node_modules/.bin/mocha --reporter=dot"
+    "unittest": "npm run build && ./node_modules/.bin/mocha --reporter=dot",
+    "build": "tsc",
+    "watch": "npm run build -- -w",
+    "prepare": "npm run build"
```

Comment:
`prepare` runs on local `npm install` as well, which currently broke my installation when jumping between branches.

Any reason for not onlyrunning in `prepack` script instead?

Reply:
Had never used `prepack` before, but looking at it I see no reason why I can't 👍